templates/esp-deployment.yaml

```diff
@@ -21,6 +21,14 @@ spec:
         app.kubernetes.io/name: esp
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
What's the rationale for using the `IgnoredDuringExecution` variant and not the `RequiredDuringExecution`? I'm not sure I would really know which one is best, but it sounds like the `IgnoredDuringExecution` would cause a pod to be killed off and rescheduled if it stopped meeting the condition... which is _think_ is probably what you want 🤷‍♀️ .... maybe scribble a note in the commit message about the reasoning

Reply:
> In the future we plan to offer requiredDuringSchedulingRequiredDuringExecution which will be just like requiredDuringSchedulingIgnoredDuringExecution except that it will evict pods from nodes that cease to satisfy the pods’ node affinity requirements.

https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature